src/components/Toast/README.md

```diff
@@ -236,7 +236,7 @@ class ToastExample extends React.Component {
       <Toast
         content="Message sent"
         onDismiss={this.toggleToast}
-        duration={4500}
+        duration={ToastDuration.Fast}
```

Comment:
I could be wrong, but it strikes me as an odd API to import the duration enum and use it this way, as props to the component it came from. Since these delay values are intended only for Toast, I would expect instead for the `duration` prop to accept the keywords directly.

For example, I would expect to write `<Toast duration="fast" />` like `<Button size="slim" />`, while still being able to write `<Toast duration={6000} />`.

Reply:
We can still export the enum—we seem to export `type Size` from `Button`, but I would expect it to be rarely imported.

Reply:
👍

Reply:
@AndrewMusgrave Is your 👍 in reference to reverting back to a number?

Reply:
Sorry I was agreeing with Ryan on the api. I would expect the usage to be `duration="fast"` or `duration={5000}` rather than forcing consumers to import out enum

Reply:
Thank you :)

Reply:
This is a good call. I don't think we use enums in other places, and instead passing in strings to the prop is how we do this in other places (e.g. `<Button size="">`)